### DISCUSSION TITLE
GIS2FEM

### DISCUSSION THREAD

**Discussion by Joaquinfnz** [mhm-ufz/mHM#77] [Q&A]
Hi, I'm working with OGS, and I need to transform the simulation variables from MHM to work with the GIS2FEM code. Does anyone know how to do it?

**Comment by rohinikumar82** [comment]
Sorry, we do not work with GIS2FEM code within mHM codebase. I could not locate someone who could help you with this issue. May be you can post this query to OGS help desk.
